### PR TITLE
Kubeflow Getting Started: add approvers and reviewers to OWNERS

### PR DESCRIPTION
**Pull request by 8bitmp3**
This PR adds a second approver in `OWNERS` for the Getting Started section.

Currently, we have 1 approver (@RFMVasconcelos) and no reviewers.

@rmgogogo @joeliedtke @bobgy @animeshsingh - if you would like to volunteer to become, for example, a third approver or a 1st and 2nd reviewer, let us know! Thanks

/assign @rmgogogo @joeliedtke @bobgy @animeshsingh

### CODE REVIEW

**Comment by 8bitmp3** [comment]
/assign @jlewi 

@jlewi If you have some time, can you help take a look at this. Thank you! 
We have a PR by @RFMVasconcelos and the repo section lists them as the only `approver`. Since there's noone else, we can't get the PR moving.

**Comment by Bobgy** [comment]
Sounds good! I can be a reviewer.

**Comment by 8bitmp3** [comment]
Thanks @Bobgy ! I've added you

**Comment by knkski** [comment]
I'd be interested in being a reviewer as well

**Comment by 8bitmp3** [comment]
@knkski I've added you as you requested.

@rmgogogo @animeshsingh @Bobgy - can you please take a look at this PR? Thank you!

File: content/en/docs/started/OWNERS [hunk 1]
```diff
@@ -1,3 +1,9 @@
 approvers:
   - RFMVasconcelos
-  
+  - 8bitmp3
+
+reviewers:
+  - Bobgy
+  - knkski 
+  - RFMVasconcelos
+  - 8bitmp3
```
Comment (Bobgy) [inline content/en/docs/started/OWNERS:9]: It's better not to duplicate reviewer and approver to make sure everyone is notified

**Comment by 8bitmp3** [comment]
@Bobgy Done! Thanks

**Comment by Bobgy** [comment]
/lgtm
/approve

Thanks

**Pull request merged**